SDL2_OpenGL_BoilerPlate/BoilerPlate/Game.cpp

```diff
@@ -0,0 +1,171 @@
+#include "Game.hpp"
+
+Game::Game() {
+
+	player = new Player();
+	numberOfAsteroids = 5.0f;
+	asteroids = std::vector<Asteroid*>(numberOfAsteroids);
+	SpawnAsteroids();
+}
+
+Game::~Game(){
+
+	RemoveFromMemory();
+}
+
+void Game::Update(int screenWidth, int screenHeight, float deltaTime) {
+
+	
+	player->Update(screenWidth, screenHeight, deltaTime);
+	UpdateAsteroids(screenWidth, screenHeight, deltaTime);
+}
+
+void Game::Render() {
+
+	player->Render();
+	RenderAsteroids();
+}
+
+void Game::UpdateAsteroids(int screenWidth, int screenHeight, float deltaTime) {
+
+	DetermineDebuggerState();
```

Comment:
I like this approach. Not the name for the function, but it's a personal thing. Overall I like the way you name stuff.